### DISCUSSION TITLE
Fine-tuning of Canary-1B model in low resource languages!

### DISCUSSION THREAD

**Discussion by nikithakriz** [NVIDIA-NeMo/NeMo#10790] [General]
Hi folks,
I am looking for cook books which shows how to fine-tune Canary-1B model on low resource Indian languages. I have no idea how dataset should be(as am new to this particular ASR models) and how to do fine-tuning. Anyone help me with this? even a small help is really appreciated!

**Comment by diarray-hub** [comment]
We are a bunch of newbies waitiing for Nvidia to publish a complete documentation along with fine tuning code examples for canary
